lib/services/dialog_import_service.rb

```diff
@@ -115,7 +123,7 @@ def build_association_list(dialog)
     dialog["dialog_tabs"].flat_map do |tab|
       tab["dialog_groups"].flat_map do |group|
         group["dialog_fields"].flat_map do |field|
-          associations << { field["name"] => field["dialog_field_responders"] } unless field["dialog_field_responders"].nil?
+          associations << { field["name"] => field["dialog_field_responders"] } if field["dialog_field_responders"].present?
```

Comment:
`if field["dialog_field_responders"]` is equivalent to `unless field["dialog_field_responders"].nil?`.  `present?` changes that a little bit.

Reply:
I think we want present, though.

Reply:
Save via the UI results in there being empty arrays if there aren't associations and we don't want them in the association list if they're empty.